pathspider/cmd/measure.py

```diff
@@ -95,7 +96,7 @@ def run_measurement(args):
         threading.Thread(target=job_feeder, args=(args.input, spider)).start()
 
         with open(args.output, 'w') as outputfile:
-            logger.info("opening output file "+args.output)
+            logger.info("opening output file "+ args.output)
```

Comment:
We usually have a space before and after the +. Either add both spaces, or add none.